ui/components/ui/button/buttons.scss

```diff
@@ -13,7 +13,7 @@
   justify-content: center;
   align-items: center;
   box-sizing: border-box;
-  border-radius: 6px;
+  border-radius: 12px;
```

Comment:
Matches `BorderRadius.XL` which is set in component library `Button` border radius update https://github.com/MetaMask/metamask-extension/pull/31818/files?file-filters%5B%5D=.tsx&show-viewed-files=true#diff-6a6f56e439ab2ab776db250936c8bb3f6ae127824f26022e5a9ca61590411c60R89